Resources/Prototypes/Entities/Items/Clothing/cloaks.yml

```diff
@@ -0,0 +1,106 @@
+- type: entity
+  parent: Clothing
+  id: CloakClothing
+  abstract: true
+  name: cloak
+  description:
+  components:
+  - type: Sprite
+    sprite: Clothing/cloak.rsi
+  - type: Icon
+    sprite: Clothing/cloak.rsi
+  - type: Clothing
+    Size: 10
+    QuickEquip: true
+    Slots:
+    - back
+    sprite: Clothing/cloak.rsi
+
+- type: entity
+  parent: CloakClothing
+  id: CapCloak
+  name: captain's cloak
+  description:
+  components:
+  - type: Sprite
+    state: capcloak
+  - type: Icon
+    state: capcloak
+  - type: Clothing
+    HeldPrefix: capcloak
+
+- type: entity
+  parent: CloakClothing
+  id: HosCloak
+  name: hos's cloak
```

Comment:
Shouldn't HOS be upper case since it's an acronym? Same for QM, CE, CMO, etc...

Reply:
I believe the standard is actually HoS and HoP for those particular acronyms, but the rest are all uppercase.